components/filter/test/filter.test.js

```diff
@@ -29,16 +29,18 @@ describe('d2l-filter', () => {
 	});
 
 	describe('loading', () => {
```

Comment:
Enhanced the loading test because we can't easily test that with visual diff, but the rest will be tested in the visual diff tests or is tested in the selection component tests